js/src/components/Metabox.js

```diff
@@ -34,5 +44,6 @@ export default function Metabox( { isContentAnalysisActive, isKeywordAnalysisAct
 Metabox.propTypes = {
 	isContentAnalysisActive: PropTypes.bool,
 	isKeywordAnalysisActive: PropTypes.bool,
```

Comment:
Isn't the rendering of the snippet editor conditional based on a prop like the other two?